src/main/core/API/Finder/Log/MessageLogFinder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\CoreBundle\API\Finder\Log;
+
+use Claroline\AppBundle\API\Finder\AbstractFinder;
+use Claroline\CoreBundle\Entity\Log\MessageLog;
+use Doctrine\ORM\QueryBuilder;
+
+class MessageLogFinder extends AbstractFinder
+{
+    public function getClass()
+    {
+        return MessageLog::class;
+    }
+
+    public function configureQueryBuilder(QueryBuilder $qb, array $searches = [], array $sortBy = null, array $options = ['count' => false, 'page' => 0, 'limit' => -1])
+    {
+        foreach ($searches as $filterName => $filterValue) {
+            switch ($filterName) {
+                case 'workspace':
```

Comment:
This prop does not exist in your entity.

Reply:
Change done, thx :)